swatch-core/src/test/java/org/candlepin/subscriptions/clowder/ClowderJsonPathPropertySourceTest.java

```diff
@@ -279,7 +122,9 @@ void testKafkaBrokersPropertyWhenNoBrokersConfiguration() throws Exception {
         "clowder.endpoints.index-service.trust-store-path|file:/tmp/truststore.*.trust",
         "clowder.endpoints.index-service.trust-store-password|.+",
         "clowder.endpoints.index-service.trust-store-type|PKCS12",
-        "clowder.privateEndpoints.export-service-service.url|http://export-service-service.svc:10000"
+        "clowder.privateEndpoints.export-service-service.url|http://export-service-service.svc:10000",
```

Comment:
Just for complete coverage, would you mind adding

```
        "clowder.endpoints.rbac-service.url|http://rbac-service.rhsm.svc:8000",
```

to the test case here?

Reply:
I don't see how adding this scenario will increase the coverage. I guess you're asking this because the rbac endpoint is not used, if so, I rather remove it.

Reply:
Just because `endpoints` and `privateEndpoints` have slightly different code paths.  `getPrivateEndpointConfig` vs `getEndpointConfig`